DMX.Gatekeeper.Api.Infrastructure.Build/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using ADotNet.Clients;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.SetupDotNetTaskV1s;
+
+var githubPipeline = new GithubPipeline
+{
+    Name = "Gatekeeper Build",
+
+    OnEvents = new Events
+    {
+        Push = new PushEvent
+        {
+            Branches = new string[] { "main" }
+        },
+
+        PullRequest = new PullRequestEvent
+        {
+            Branches = new string[] { "main" }
+        }
+    },
+
+    Jobs = new Jobs
+    {
+        Build = new BuildJob
+        {
+            RunsOn = BuildMachines.Windows2022,
+
+            Steps = new List<GithubTask>
+            {
+                new CheckoutTaskV2
+                {
+                    Name = "Checking out Code"
+                },
+
+                new SetupDotNetTaskV1
+                {
+                    Name = "Installing .NET",
+
+                    TargetDotNetVersion = new TargetDotNetVersion
+                    {
+                        DotNetVersion = "7.0.100-preview.4.22252.9",
+                        IncludePrerelease = true
+                    }
+                },
+
+                new RestoreTask
+                {
+                    Name = "Restoring Packages"
+                },
+
+                new DotNetBuildTask
+                {
+                    Name = "Building Project(s)"
+                },
+
+                new TestTask
+                {
+                    Name = "Running Tests"
+                }
+            }
+        }
+    }
+};
+
+var adotNetClient = new ADotNetClient();
+adotNetClient.SerializeAndWriteToFile(
```

Comment:
Separate this multi-liner liner with an empty line.

here's a rule:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations